### PR TITLE
Make zask Pep8 compatible and add Travis to check

### PR DESCRIPTION
**Pull request by wanleung**
## Summary of changes

add .travis.yml for the Travis settings
formatted the code to pass pep8
## How to Test

pep8 ./zask ./tests
Travis will work after the merge

/fyi: @tjoelsson , @imlazyone : Please review.

### CODE REVIEW

**Comment by KixPanganiban** [comment]
@wanleung By any chance, can we squash some of these commits? `testing`, `testing`, `typo` don't look too good. 😛

**Comment by stuartwan** [comment]
@KixPanganiban I don't really have time to review this, I'll assign this to you.

**Comment by KixPanganiban** [comment]
Sure @imlazyone.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+language: python
+
+python:
+  - "2.7"
+
+install:
+    - pip install pep8
+
+before_script:
+    - "pep8 ./zask ./tests"
```
Comment (tjoelsson) [inline .travis.yml]: ? I think you can also check the bin/ folder.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+script:
+    - python setup.py test
```
Comment (tjoelsson) [inline .travis.yml]: this will pass?

Comment (tjoelsson) [inline .travis.yml]: It doesn't do anything. Shouldn't it run the unit tests?

File: zask/__init__.py [hunk 3]
```diff
@@ -40,10 +40,14 @@ class LocalContext(object):
     """
     Store data in local greenlet context.
 
-    > Gevent also allows you to specify data which is local to the greenlet context.
-    > Internally, this is implemented as a global lookup which addresses a private namespace keyed by the greenlet's getcurrent() value.
+    > Gevent also allows you to specify data which is local to
+      the greenlet context.
+    > Internally, this is implemented as a global lookup
+      which addresses a private namespace keyed by the
+      greenlet's getcurrent() value.
```
Comment (tjoelsson) [inline zask/__init__.py:49]: This doesn't render correctly when building sphinx documentation:

screen shot 2016-08-02 at 6 25 01 pm

**Comment by arkhi** [comment]
This PR has not seen any activity in the last 15 days; we should close it or work on it. We can reopen it when necessary.

Thanks!

**Comment by wanleung** [comment]
The unit test is now working after merge the latest master which the zack version updated to 1.9.4

**Comment by wanleung** [comment]
@KixPanganiban @tjoelsson Please review.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+language: python
+
+python:
+  - "2.7"
+
+install:
+    - pip install pep8 tox
+
+before_script:
+    - "pep8 --ignore=E123,E402,E501,E701 ./bin ./zask ./tests"
```
Comment (KixPanganiban) [inline .travis.yml]: I get why we're skipping E402 (module import not at top of file) for reasons, but why are we skipping the following?

| code | description |
| --- | --- |
| E128 | continuation line under-indented for visual indent |
| E501 | line too long (82 > 79 characters) |
| E701 | multiple statements on one line (colon) |

This somewhat defeats the purpose of Travis and doesn't make the repo truly PEP8-compatible.

File: .travis.yml [hunk 5]
```diff
@@ -5,13 +5,13 @@ python:
   - "2.7"
 
 install:
-    - pip install pep8
+    - pip install pep8 tox
 
 before_script:
-    - "pep8 ./zask ./tests"
+    - "pep8 --ignore=E123,E402,E501,E701 ./bin ./zask ./tests"
```
Comment (tjoelsson) [inline .travis.yml]: When I run `pep8 ./bin ./zask ./tests` I only see E402 errors. Is it not enough to ignore that one?

**Comment by tjoelsson** [comment]
👍

**Comment by KixPanganiban** [comment]
👍

**Pull request merged**